CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing Guidelines
+
+
+## Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+
+
+
+## Getting Started
+
+0. [Fork the repo!](https://github.com/rsm23/ncs-jss.github.io/new/master#fork-destination-box)
```

Comment:
Repo link should be the link of this repo rather than the link of your fork repo